README.md

```diff
@@ -11,7 +11,8 @@ CocoaPods manages dependencies for your Xcode projects.
 You specify the dependencies for your project in a simple text file: your `Podfile`. 
 CocoaPods recursively resolves dependencies between libraries, fetches 
 source code for all dependencies, and creates and maintains an Xcode 
-workspace to build your project.
+workspace to build your project. The latest released Xcode versions and the 
```

Comment:
I would add the word "Typically, ...` atm unfortunately we never guarantee this nor do we test for it.

Reply:
This word change can be a default commitment. If things are not working as described here then they are a bug which should be reported.